tigerpath/majors_and_certificates/majors/SPO_2018.json

```
@@ -0,0 +1,359 @@
+              "course_list": [
+                "SPA 205",
+                "SPA 207",
+                "SPA 207S",
+                "SPA 208S",
+                "SPA 209",
+                "SPA 307"
+              ]
+            },
+            {
+              "name": "Pre-1800 Literature",
+              "max_counted": null,
+              "min_needed": null,
+              "explanation": "<i>Editor's Note: This requirement is too broad, so TigerPath does not check it.</i>\n\nAll Spanish concentrators must take one course in pre-1800 literature. University regulations limit to 12 the number of departmental courses allowed to each student in his or her concentration. Students cannot earn both a SPO concentration (in any track) and a SPO certificate (in any language). Students interested in focusing both in Spanish and Portuguese may choose Track 2 for their concentration.",
+              "no_req": null
+            },
+            {
+              "name": "Upper Division Courses",
+              "max_counted": 1,
+              "min_needed": 5,
+              "explanation": "Any track in the concentration in Spanish and/or Portuguese Literature and Cultures requires a minimum of eight upper-division courses, at least five of which must be in the language of concentration. With the approval of the Director of Undergraduate Studies, up to three courses taken during a semester abroad may count towards the concentration if the student is pursuing Track 1 or Track 2. Up to two courses taken during a semester abroad may be counted towards the concentration if the student is pursuing Track 3 or Track 4.",
+              "course_list": [
+                "SPA 2*",
+                "SPA 3*",
+                "SPA 4*",
+                "SPA 5*"
+              ],
+              "excluded_course_list": [
+                "SPA 20*"
+              ]
+            },
+            {
+              "name": "Tracks",
+              "max_counted": 1,
+              "min_needed": 1,
+              "explanation": "Departmental courses cover a wide array of literary, cultural, social, historical, and political topics. Students are, therefore, able to pursue courses of study that are tailor-made to their own individual interests. The department offers four different tracks for concentrators.",
+              "req_list": [
+                {
+                  "name": "One Language, Literature, and Culture",
+                  "max_counted": 1,
+                  "min_needed": 3,
+                  "explanation": "Concentration in one language, literature, and cultures (Spanish or Portuguese). Eight upper-division courses in the language of concentration. Up to two of those courses may be taken in English, in which case all written work must be completed in the target language.",
+                  "course_list": [
+                    "SPA 2*",
+                    "SPA 3*",
+                    "SPA 4*"

```

Why not also 500-level here?
```suggestion
                    "SPA 4*",
                    "SPA 5*"
```